app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
+    total_rating = 0
```

Comment:
Attempt to perform this calculation in a single line by using a generator expression and the `sum `function